### DISCUSSION TITLE
Cannot play this Replay

### DISCUSSION THREAD

**Discussion by bjlake** [fraxiinus/ReplayBook#238] [Troubleshooting]
Replay book used to work for me, but not although it can detect all the games and stats the play is grayed out and says "cannot play this replay."

i thought it might only be new files but its all the replays.

**Comment by fraxiinus** [comment]
Check what executables are registered in ReplayBook

https://www.fraxiinus.dev/ReplayBook/troubleshooting/play-button-disabled/

**Reply by bjlake** [reply]
that was it! thanks a bunch!

**Reply by bjlake** [reply]
I spoke too soon, now its allowing me to start it but its just loading indefinitely

**Reply by fraxiinus** [reply]
stuck loading might be a locale set wrong. Try giving this a shot https://www.fraxiinus.dev/ReplayBook/troubleshooting/issues-playing-replays/#how-to-change-an-executables-locale

**Reply by bjlake** [reply]
no dice, just indefinite black screen. no error message

**Reply by fraxiinus** [reply]
Very strange... If you haven't already try all the suggestions in the troubleshooting page. Besides that I'm out of ideas to test.
